zhaquirks/xiaomi/aqara/switch_acn047.py

```diff
@@ -206,5 +207,12 @@ class AqaraT2Relay(XiaomiCustomDevice):
     }
 
     device_automation_triggers = {
-        (SHORT_PRESS, SHORT_PRESS): {COMMAND: SINGLE},
+        (SHORT_PRESS, PRESS_TYPES[1]): {
```

Comment:
Would `button_1` and `button_2` also fit? (from [zhaquirks/const.py#L20-L21](https://github.com/zigpy/zha-device-handlers/blob/6ab4376f4947c456a1f1ba05e9218519bc970100/zhaquirks/const.py#L20-L21)) 
They should have a translation in the UI: [homeassistant/components/zha/strings.json#L228-L252](https://github.com/home-assistant/core/blob/5dac5d5c7e9c5a1bf3ec4edc5198a7a0d92d27ee/homeassistant/components/zha/strings.json#L228-L252)

Reply:
I am not sure. Maybe.
<img width="518" alt="Screenshot 2024-01-27 at 8 32 17" src="https://github.com/zigpy/zha-device-handlers/assets/1564933/b37454ac-c3a3-4dc8-bf16-333bd587e1a6">
It is SW1-COM and SW2-COM shorted.

Reply:
The best would be to not so send events IMHO but to have binary sensors if these are shorted or not. This device is a perfect fit for garage doors but this way it requires extra automation and sensor implementation in HA.

Another strange thing:
- When it is in toggle mode then it sends two events when I close circuit and when I open.
- When it is in momentary then it sends one event when I open the circuit.

So it is not very optimal. Is it something can be handled from the quirk or it is the device's firmware?

Reply:
The device sends events instead of exposing an `OnOff` cluster for example.
Since push buttons (momentary mode) only send one event, a binary sensor wouldn't make sense, as we'd have to artificially turn off the signal after a delay.
Like, you can do many things with a quirk, but we shouldn't do this probably.

I'd just set up an automation in HA with events from the relay (with different trigger ids) that toggle `input_boolean` switches. Something like that.

Also, let's use the `button_1` and `button_2` constants here, so it's properly translated in the HA UI. And it's also better to not confuse it with the switch / relay part of the device.